packages/v4/patternfly-docs/content/design-guidelines/components/navigation/navigation.md

```diff
@@ -17,7 +17,8 @@ A **navigation** organizes an application’s structure and content, making it e
 5. [**Secondary horizontal navigation**](#secondary-horizontal-navigation)
 
 
```

Comment:
Feel like there is a "Variations" heading missing for these navigation guidelines (?)  Or maybe instead of "Usage" it should say "Variations" - or this could be similar to labels where we need to add a general "Usage" section to describe when to use a navigation (can be a one liner)